### PR TITLE
J16bit youtube gui bits

### PR DESCRIPTION
**Pull request by Jenna64bit**
Lets be honest, jq is all that matters here

### CODE REVIEW

File: packages/jq/SPECS/jq.spec [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+%{_datadir}/doc/jq/README
+%{_datadir}/doc/jq/README.md
+
+%files devel
+%{_includedir}/jq.h
+%{_includedir}/jv.h
+%{_libdir}/libjq.so
+
+
+%changelog
```
Comment (unxmaal) [inline packages/jq/SPECS/jq.spec:90]: pls update the changelog

File: packages/jq/SPECS/jq.spec [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+%prep
+%setup -qn %{name}-%{version}
+
+%build
+export CFLAGS="-I%{_includedir}/libdicl-0.1 -D_SGI_SOURCE -D_SGI_REENTRANT_FUNCTIONS $RPM_OPT_FLAGS"
+export CXXFLAGS="-I%{_includedir}/libdicl-0.1 -D_SGI_SOURCE -D_SGI_REENTRANT_FUNCTIONS $RPM_OPT_FLAGS"
+export LDFLAGS="-ldicl-0.1 $RPM_LD_FLAGS -lgen"
+%configure --disable-static
+make %{?_smp_mflags}
+# Docs already shipped in jq's tarball.
```
Comment (unxmaal) [inline packages/jq/SPECS/jq.spec:49]: if we have the origfedora we don't need commented-out code

File: packages/yad/SPECS/yad.spec [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+
+BuildRequires:  gcc
+BuildRequires:  gtk2-devel >= 2.20.1
+BuildRequires:  desktop-file-utils
+BuildRequires:  perl(XML::Parser)
+BuildRequires:  intltool >= 0.40.0
+BuildRequires:  pkgconfig
+BuildRequires:  gettext
+
```
Comment (unxmaal) [inline packages/yad/SPECS/yad.spec:18]: does yad need jq and onigurama too?

**Review by unxmaal — CHANGES_REQUESTED** [review]

File: packages/yad/SPECS/yad.spec (hunk 3 above)
Comment (Jenna64bit) [reply packages/yad/SPECS/yad.spec:18]: nope, jq needs onigurama though. yad is otherwise unrelated excepting I needed both in the dumb youtube thing

**Review by unxmaal — APPROVED** [review]

**Pull request merged**